### ISSUE TITLE
questions about hw2 csv file.

### ISSUE THREAD

**Issue by rkdgnlgus2** [swsnu/swppfall2021#26]
Hello, I have a question about babyname CSV file. I already asked whether csv file always has header and the answer was yes.
Then is the header always given as (year,rank,name,gender,rankChange)?
I mean, does each row of csv datas always have form of an example (2008,219,Harrison,M,-6)?

**Comment by kdh0102** [comment]
Yes, each row will have values from each column(optional for the `rankChange`) in order, and a comma will separate each value.

**Comment by rkdgnlgus2** [comment]
Thank you!
